streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -773,7 +774,7 @@ public void clear() {
 
         try {
             restoreConsumer.unsubscribe();
-        } catch (KafkaException e) {
+        } catch (final KafkaException e) {
```

Comment:
Why did this not fail the build without the `final` qualifier? Do we have some exceptions for this style check that we should remove?

Reply:
This file was not in any suppression files.. I did not know why it does not fail the build before.